internal/cli/performanceadvisor/suggestedindexes/list.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package suggestedindexes
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const listTemplate = `ID	NAMESPACE	INDEX{{range .SuggestedIndexes}}
+{{- $id := .ID}}   {{- $name := .Namespace}}  
+{{ $id }}	{{ $name }}	{	{{- range $i, $element := .Index}}
+{{- range $key, $value := .}} {{if $i }},{{end}} "{{ $key }}":{{ $value }}{{ end }}{{ end }}  }{{ end }}
+`
```

Comment:
[nit] This is what we want ell done 💯 , just a couple of things, first the spacing seems odd, this in particular can be a problem as the tabbing function is going to mess the output, `{	{{- range $i` it looks you're using a tab and not a space, a tab means a new column and this is all part of the same column

Another thing is the assignment of ID and Namespace to some variables this feels like it just adds noice
And finally we can save some space if we remove the quotes we can save some space, specially since they are not needed here

You can have this be
```suggestion
const listTemplate = `ID	NAMESPACE	INDEX{{range .SuggestedIndexes}}  
{{ .ID }}	{{ .Namespace}}	{ {{range $i, $element := .Index}}{{range $key, $value := .}}{{if $i}}, {{end}}{{ $key }}: {{ $value }}{{end}}{{end}} }{{end}}
`
```

And it will output like 
```bash
ID                         NAMESPACE            SUGGESTED INDEX
5f69cf3e622c776f5c4d04d9   loadTest.saildrone   { timeUTC: 1 }
5f69cf3e622c776f5c4d04da   loadTest.saildrone   { id: 1, arrayOperation: 1, decimal: 1, nested.field: 1, quotedReservedWordField: 1, date: 1, binary: 1, embeddedArrayDocument: 1, embeddedDocument: 1, embeddedArray: 1, embeddedDocumentArray: 1, nullValue: 1, coordinates.latitude: 1 }
5f69cf3e622c776f5c4d04db   loadTest.saildrone   { RH_MEAN: 1 }
5f69cf3e622c776f5c4d04dc   loadTest.saildrone   { trajectory: 1 }
```